indexer/services/ender/src/helpers/postgres/postgres-functions.ts

```diff
@@ -26,41 +26,47 @@ function newScript(name: string, scriptPath: string): PostgresFunction {
   };
 }
 
-const scripts: string[] = [
-  'create_extension_pg_stat_statements.sql',
-  'create_extension_uuid_ossp.sql',
+const HANDLER_SCRIPTS: string[] = [
   'dydx_asset_create_handler.sql',
-  'dydx_block_processor.sql',
   'dydx_block_processor_ordered_handlers.sql',
   'dydx_block_processor_unordered_handlers.sql',
-  'dydx_clob_pair_status_to_market_status.sql',
   'dydx_deleveraging_handler.sql',
+  'dydx_funding_handler.sql',
+  'dydx_liquidity_tier_handler.sql',
   'dydx_market_create_handler.sql',
   'dydx_market_modify_handler.sql',
   'dydx_market_price_update_handler.sql',
+  'dydx_perpetual_market_handler.sql',
+  'dydx_stateful_order_handler.sql',
+  'dydx_subaccount_update_handler.sql',
+  'dydx_transfer_handler.sql',
+  'dydx_update_clob_pair_handler.sql',
+  'dydx_update_perpetual_handler.sql',
+];
+
+const HELPER_SCRIPTS: string[] = [
+  'create_extension_pg_stat_statements.sql',
+  'create_extension_uuid_ossp.sql',
```

Comment:
Could we use a separate list like DB_SETUP_SCRIPTS for these as they are used for configuration?

Reply:
which scripts do you mean by "these"?